src/services/types.ts

```diff
@@ -221,6 +221,12 @@ namespace ts {
         durationMs: number;
     }
 
+    export enum LanguageServiceMode {
+        Semantic,

```

Comment:
Currently this is number enum, should this be string enum so we can directly match strings in `--serverMode` and pass it down?

Reply:
I don't feel strongly one way or the other.